pyodide-build/pyodide_build/buildall.py

```diff
@@ -457,9 +457,16 @@ def generate_packages_json(
     output_dir: Path, pkg_map: dict[str, BasePackage]
 ) -> dict[str, dict[str, Any]]:
     """Generate the package.json file"""
+
+    import sys
+
+    sys.path.append(str(common.get_pyodide_root() / "src/py"))
+    from pyodide import __version__
```

Comment:
I'm not sure that this package would always be importable on the host in the future, but we can deal with that issue when it happens.